### PR TITLE
Add Windows instructions to README.md

### PR DESCRIPTION
**Pull request by pjanotti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
-# SignalFx Tracing Library for .NET Core on Linux
+# SignalFx Tracing Library for .NET
 
-The SignalFx Tracing Library for .NET Core on Linux provides an
+The SignalFx Tracing Library for .NET provides an
 OpenTracing-compatible tracer and automatically configured instrumentations
-for popular .NET Core libraries and frameworks.  It supports .NET Core 2.0+ on
-Linux distributions.
+for popular .NET libraries and frameworks.  It supports .NET Core 2.0+ on
+Linux and Windows and .NET Framework 4.5+ on Windows.
```
Comment (rmfitzpatrick) [inline README.md:6]: Not sure official support for framework is intended atm.  Instrumentations are provided under beta status?

Comment (pjanotti) [reply README.md:6]: Yes, we are adding Framework support too. The beta status for the instrumentation is mentioned below.

File: README.md [hunk 2]
```diff
@@ -71,10 +73,38 @@ manager:
     $ dotnet run
     ```
 
+### Windows
+
+1. Install the CLR Profiler using the installers (MSI files) from
+[latest release](https://github.com/signalfx/signalfx-dotnet-tracing/releases/latest),
+choose the installer (x64 or x86) according to the architecture of the application
+to be instrumented.
```
Comment (brianashby-sfx) [inline README.md:81]:
```suggestion
1. Install the CLR Profiler using an installer file (`.msi` file) from the
[latest release](https://github.com/signalfx/signalfx-dotnet-tracing/releases/latest).
Choose the installer (x64 or x86) according to the architecture of the application
you're instrumenting.
```

File: README.md [hunk 3]
```diff
@@ -71,10 +73,38 @@ manager:
...
+2. Configure the required environment variables:
```
Comment (brianashby-sfx) [inline README.md:82]: Steps 2-4 involve setting environment variables. For step 2, can we be more specific about which environment variables a user is setting?

File: README.md [hunk 4]
```diff
@@ -71,10 +73,38 @@ manager:
...
+   ```
+3. Set the service name:
+   ```batch
+   set SIGNALFX_SERVICE_NAME=MyServiceName
+   ```
+4. Set the endpoint of a Smart Agent or OpenTelemetry Collector:
+   ```batch
+   set SIGNALFX_ENDPOINT_URL='http://<YourSmartAgentOrCollector>:9080/v1/trace'
+   ```
+5. Re-start your application ensuring that all environment variables are properly
```
Comment (brianashby-sfx) [inline README.md:101]:
```suggestion
5. Restart your application ensuring that all environment variables are properly
```

Comment (brianashby-sfx) [reply README.md:101]: What should a user do to ensure that all environment variables are properly configured?

Comment (pjanotti) [reply README.md:101]: AFAIK Windows doesn't have a direct way to do this I will suggest a tool that can help with that - the tool is well-known for Windows users.

**Pull request merged**